code/Language/Drasil/Printing/Import.hs

```diff
@@ -261,7 +261,7 @@ createLayout sm = map (sec sm 0)
 
 -- | Helper function for creating sections at the appropriate depth
 sec :: HasSymbolTable ctx => ctx -> Int -> Section -> T.LayoutObj
-sec sm depth x@(Section title contents _) =
+sec sm depth x@(Section title contents _ _) = --FIXME: should ShortName be used somewhere?
```

Comment:
Yes, something here needs changed. But it can be done later. @Mornix is building expertize on this, so he's probably in the best place to do so.  Please file an issue though.